artifacts/oopsla25-bv-decide/makefile

```diff
@@ -4,10 +4,8 @@ artifact.pdf: artifact.tex sigplanconf.cls
 	latexmk -pdf -shell-escape artifact.tex
 
 docker:
-	docker build -t lean-image
-	docker create --name temp-container lean-image
-	docker cp temp-container:/code/lean-mlir/bv-evaluation/raw-data ./docker-raw-results
-	docker rm temp-container
+	podman build -t oopsla25-bv-decide --progress plain .
+	podman save -o ./oopsla25-bv-decide.tar oopsla25-bv-decide
```

Comment:
Why the switch from podman to docker? The latter seems more standard: even if the user has podman installed, it's common to provide a docker-compatible alias, but I haven't heard of docker providing a podman alias.